### DISCUSSION TITLE
Auto or Manual Resistance for Domyos Elliptical

### DISCUSSION THREAD

**Discussion by eko136** [cagnulein/qdomyos-zwift#431] [Q&A]
Hi, there.
Thank you for creating this qDomyos, I managed to do Zwifting with my existing Domyos elliptical trainer.

This may have been discussed in the previous posts, is there any future development to make the auto resistance for Domyos elliptical trainer. I know this is challenging to associate the resistance in Zwift with the resistance scaling in various elliptical trainers.

Or, is there any development to enable the manual resistance buttons on the trainer to work to control the resistance?
Currently, if I want to change the resistance manually, I have to do it with my bridge phone and it is really difficult pressing them in the phone while I am running.

Thank you again for making this happen.

Best regards,

Eko

**Comment by cagnulein** [comment]
Hi, i can do easility the same thing that i did for the other elliptical
that i manage. So you will have to do bike zwifting and then the resistance
will be auto controlled. are you ok with that?

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno sab 28 ago 2021 alle ore 05:59 eko136 ***@***.***>
ha scritto:

**Comment by eko136** [comment]
Dear Roberto,
I am not sure I am following you.Does that mean I will pick the Ride option, instead of Run, with the avatar riding a bicycle but controlled by me running the elliptical? (CMIIW)That would be awkward, but interesting.

I have tried it just now. Power and Controllable can detect my Domyos, but no power reading detected although qdomyos showed some power reading. 
Anything that I missed from the setting? Please assist. 
Thank you.
Best regards, 

Eko Arief Wicaksono
    On Saturday, August 28, 2021, 01:55:30 PM GMT+3, Roberto Viola ***@***.***> wrote:  
 
 

Hi, i can do easility the same thing that i did for the other elliptical
that i manage. So you will have to do bike zwifting and then the resistance
will be auto controlled. are you ok with that?

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno sab 28 ago 2021 alle ore 05:59 eko136 ***@***.***>
ha scritto:

> Hi, there.
> Thank you for creating this qDomyos, I managed to do Zwifting with my
> existing Domyos elliptical trainer.
>
> This may have been discussed in the previous posts, is there any future
> development to make the auto resistance for Domyos elliptical trainer. I
> know this is challenging to associate the resistance in Zwift with the
> resistance scaling in various elliptical trainers.
>
> Or, is there any development to enable the manual resistance buttons on
> the trainer to work to control the resistance?
> Currently, if I want to change the resistance manually, I have to do it
> with my bridge phone and it is really difficult pressing them in the phone
> while I am running.
>
> Thank you again for making this happen.
>
> Best regards,
>
> Eko
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cagnulein/qdomyos-zwift/discussions/431>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAALYWCJ6UVLLE6QKG2L6HLT7BNLTANCNFSM5C6VZBUA>
> .
>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by cagnulein** [comment]
Yes it’s what I mean but I have to do the modification first :)
Zwift can control the resistance only in bike mode, that’s why we need to
do this trick ;)
I will do it tomorrow
R.

Il giorno sab 28 ago 2021 alle 22:05 eko136 ***@***.***> ha
scritto:

> Dear Roberto,
> I am not sure I am following you.Does that mean I will pick the Ride
> option, instead of Run, with the avatar riding a bicycle but controlled by
> me running the elliptical? (CMIIW)That would be awkward, but interesting.
>
> I have tried it just now. Power and Controllable can detect my Domyos, but
> no power reading detected although qdomyos showed some power reading.
> Anything that I missed from the setting? Please assist.
> Thank you.
> Best regards,
>
> Eko Arief Wicaksono
> On Saturday, August 28, 2021, 01:55:30 PM GMT+3, Roberto Viola
> ***@***.***> wrote:
>
>
>
> Hi, i can do easility the same thing that i did for the other elliptical
> that i manage. So you will have to do bike zwifting and then the resistance
> will be auto controlled. are you ok with that?
>
> Roberto Viola
> Software engineer and open source enthusiast
> http://robertoviola.cloud
>
>
>
> Il giorno sab 28 ago 2021 alle ore 05:59 eko136 ***@***.***>
> ha scritto:
>
> > Hi, there.
> > Thank you for creating this qDomyos, I managed to do Zwifting with my
> > existing Domyos elliptical trainer.
> >
> > This may have been discussed in the previous posts, is there any future
> > development to make the auto resistance for Domyos elliptical trainer. I
> > know this is challenging to associate the resistance in Zwift with the
> > resistance scaling in various elliptical trainers.
> >
> > Or, is there any development to enable the manual resistance buttons on
> > the trainer to work to control the resistance?
> > Currently, if I want to change the resistance manually, I have to do it
> > with my bridge phone and it is really difficult pressing them in the
> phone
> > while I am running.
> >
> > Thank you again for making this happen.
> >
> > Best regards,
> >
> > Eko
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/cagnulein/qdomyos-zwift/discussions/431>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AAALYWCJ6UVLLE6QKG2L6HLT7BNLTANCNFSM5C6VZBUA
> >
> > .
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.
> Triage notifications on the go with GitHub Mobile for iOS or Android.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/cagnulein/qdomyos-zwift/discussions/431#discussioncomment-1248352>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAALYWB3BZJCIR33NSHH65DT7E6RDANCNFSM5C6VZBUA>
> .
>
-- 
Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud

**Comment by cagnulein** [comment]
@eko136 it's in the beta right now. Remember to enable the "treadmill as a bike setting"

**Comment by eko136** [comment]
Dear Roberto,
Yes, I noticed the beta version, but I think you assign it to my Google account ***@***.***, I also purchased your app in my Apple account ***@***.*** cannot see it in that Apple account.
If you still remember, I re-purchased your app in Apple because my Honor 8X does not have the capability to re-broadcast.Please also assign the beta tester to my Apple account ***@***.*** also.
Thank you.
Best regards,

Eko
 

    On Monday, August 30, 2021, 11:10:10 AM GMT+3, Roberto Viola ***@***.***> wrote:  
 
 


@eko136 it's in the beta right now. Remember to enable the "treadmill as a bike setting"

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by cagnulein** [comment]
I can’t see your emails here, send me an email directly:)

Il giorno lun 30 ago 2021 alle 22:25 eko136 ***@***.***> ha
scritto:

> Dear Roberto,
> Yes, I noticed the beta version, but I think you assign it to my Google
> account ***@***.***, I also purchased your app in my Apple account
> ***@***.*** cannot see it in that Apple account.
> If you still remember, I re-purchased your app in Apple because my Honor
> 8X does not have the capability to re-broadcast.Please also assign the beta
> tester to my Apple account ***@***.*** also.
> Thank you.
> Best regards,
>
> Eko
>
>
> On Monday, August 30, 2021, 11:10:10 AM GMT+3, Roberto Viola ***@***.***>
> wrote:
>
>
>
>
> @eko136 it's in the beta right now. Remember to enable the "treadmill as a
> bike setting"
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.
> Triage notifications on the go with GitHub Mobile for iOS or Android.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/cagnulein/qdomyos-zwift/discussions/431#discussioncomment-1256042>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAALYWDCOD55UYHM42CXX63T7PSKTANCNFSM5C6VZBUA>
> .
>
-- 
Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud
